docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+if [[ ! -d "$DATA_DIR" ]]; then
+        exit 1
```

Comment:
Does this ever happen? I mean sure, if you don't mount the directory, but we do mount it. I think you can remove this check, let it crash later naturally when doing `migrate`.